src/main/java/at/hannibal2/skyhanni/features/inventory/OldSkyblockMenu.kt

```diff
@@ -50,8 +50,8 @@ object OldSkyblockMenu {
         if (!isEnabled()) return
 
         val sbButton = slotMap[event.slot]?.takeIf { !it.disabled } ?: return
-        val isAlreadySbButton = event.originalItem.displayName.endsWith(sbButton.displayName)
-        if (isAlreadySbButton) return
+        val isAlreadySbButton = event.originalItem?.displayName?.endsWith(sbButton.displayName)
+        if (isAlreadySbButton == true) return
```

Comment:
```suggestion
        val isAlreadySbButton = event.originalItem?.displayName?.endsWith(sbButton.displayName) == true
        if (isAlreadySbButton) return
```
Nitpick, but this makes more sense.